i6_models/assemblies/conformer/conformer_v2.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
+
+__all__ = ["ConformerBlockV2Config", "ConformerEncoderV2Config", "ConformerBlockV2", "ConformerEncoderV2"]
+
+import torch
+from torch import nn
+from dataclasses import dataclass
+from typing import Tuple, List, Dict
+
+from i6_models.config import ModelConfiguration, ModuleFactoryV1
+from i6_models.parts.conformer import (
+    ConformerConvolutionV1,
+    ConformerConvolutionV1Config,
+    ConformerMHSAV1,
+    ConformerMHSAV1Config,
+    ConformerPositionwiseFeedForwardV1,
+    ConformerPositionwiseFeedForwardV1Config,
+)
+
+
+@dataclass
+class ConformerBlockV2Config(ModelConfiguration):
+    """
+    Attributes:
+        ff_cfg: Configuration for ConformerPositionwiseFeedForwardV1
+        mhsa_cfg: Configuration for ConformerMHSAV1
+        conv_cfg: Configuration for ConformerConvolutionV1
+        swap_mhsa_conv: swap the execution order of MHSA and Conv module
+    """
+
+    # nested configurations
+    ff_cfg: ConformerPositionwiseFeedForwardV1Config
+    mhsa_cfg: ConformerMHSAV1Config
+    conv_cfg: ConformerConvolutionV1Config
+    swap_mhsa_conv: bool = True
```

Comment:
Probably better to have this `False` by default here.

Reply:
After the latest experiments I would also agree to have it false, we still do not see consistent and reliable improvements...